README.md

```diff
@@ -47,7 +47,7 @@ After generating, use appropiate tags in both `nx.json` and `.eslintrs.json` to
 
 ## Development server
 
-Run `ng serve my-app` for a dev server. Navigate to <http://localhost:4200/>. The app will automatically reload if you change any of the source files.
+Run `npm run start` for a dev server. Navigate to <http://0.0.0.0:4200/>. The app will automatically reload if you change any of the source files.
```

Comment:
I just realised that we didn't share username and password for API sandbox identity.

Reply:
Why 0.0.0.0? localhost should work fine...

Reply:
> Why 0.0.0.0? localhost should work fine...

Just checked it again. Yes. http://localhost:4200/ works!

Reply:
> Why 0.0.0.0? localhost should work fine...

Just checked it again. Yes. http://localhost:4200/ works!